### PR TITLE
wallet_rpc: add scan_tx

### PR DESCRIPTION
**Pull request by tobtoht**

### CODE REVIEW

**Comment by selsta** [comment]
bump WALLET_RPC_VERSION_MINOR

**Comment by tobtoht** [comment]
Thanks, fixed.

File: src/wallet/wallet_rpc_server.cpp [hunk 1]
```diff
@@ -3013,6 +3013,41 @@ namespace tools
...
+      {
+          cryptonote::blobdata txid_blob;
+          if(!epee::string_tools::parse_hexstr_to_binbuff(*i++, txid_blob) || txid_blob.size() != sizeof(crypto::hash))
+          {
+              er.code = WALLET_RPC_ERROR_CODE_WRONG_TXID;
+              er.message = "TX ID has invalid format";
+              return false;
+          }
+
+          crypto::hash txid = *reinterpret_cast<const crypto::hash*>(txid_blob.data());
```
Comment (moneromooo-monero) [inline src/wallet/wallet_rpc_server.cpp:3038]: hex_to_pod does the whole thing fwiw

**Review by moneromooo-monero — APPROVED** [review]

**Review by xmrnightknight — APPROVED** [review]
LGTM

**Pull request merged**